imageprocessor/views.py

```diff
@@ -108,3 +111,25 @@ def register(request):
     context['form'] = form
     return render(request, 'register.html', context)
 
+class ClassifyAPI(APIView):
```

Comment:
Is APIView being used?

Reply:
I believe that the APIView baseclass is what allows us to define a controller without needing the to annotate with csrf exemption. Also, I think it allows us to set up request authentication and request rate limiting to API in the future.

Reply:
In that case, you can probably remove the csrf_exempt annotation from the classify view. I tested it out and it seems like we don't need it anymore